staticfiles/js/requester/controllers/requester.controller.js

```diff
@@ -0,0 +1,80 @@
+/**
+* Requester Task Portfolio Controller
+* @namespace crowdsource.requester.controllers
+*/
+(function () {
+  'use strict';
+
+  angular
+    .module('crowdsource.requester.controllers')
+    .controller('RequesterTaskPortfolioController', RequesterTaskPortfolioController);
+
+  RequesterTaskPortfolioController.$inject = ['$location', '$scope', 'Authentication'];
+
+  /**
+  * @namespace HomeController
+  */
+  function RequesterTaskPortfolioController($location, $scope, Authentication) {
+    var vm = this;
+
+    $scope.user = {
+      TaskCategories : [{
```

Comment:
would <i>really</i> like this to come from the backend using an api endpoint as we have set up with other routes